tests/testthat/test-ctstm.R

```diff
@@ -614,6 +614,16 @@ test_that("IndivCtstm - joint", {
   ictstm <- IndivCtstm$new(trans_model = mstate_mix)
   disprog <- ictstm$sim_disease()$disprog_
   expect_true(is.data.table(disprog))
+
+  # Mixture
+  mstate_mix2 <- create_IndivCtstmTrans(params, input_data = msfit_data, 
```

Comment:
We could probably have better unit tests for correctness here, but my test which you followed is admittedly not great. Not a necessary update to get this merged, but more of a note for future reference for myself.

Reply:
TODO?

Reply:
Mostly. However, you should still at least test that each transition type runs and produces the expected output. Currently the test is only for the "reset" type.

Reply:
I have now (a) written some simple tests for each of the three transition types and (b) removed one line of debug code.

Is there anything further? 

Kindly, Mark.

Reply:
Looks good. Only comment is to avoid duplicating the test code with a function per the suggestion below.